frappe/email/doctype/auto_email_report/auto_email_report.py

```diff
@@ -236,5 +236,14 @@ def make_links(columns, data):
 			elif col.fieldtype == "Dynamic Link":
 				if col.options and row.get(col.fieldname) and row.get(col.options):
 					row[col.fieldname] = get_link_to_form(row[col.options], row[col.fieldname])
+			elif col.fieldtype == "Currency":
+				row[col.fieldname] = frappe.format_value(row[col.fieldname], col)
 
 	return columns, data
+
+def update_field_types(columns):
+	for col in columns:
+		if col.fieldtype in  ("Link", "Dynamic Link", "Currency")  and col.options != "Currency":
```

Comment:
```suggestion
		if col.fieldtype in ("Link", "Dynamic Link") and col.options != "Currency":
```
Do we also create link for currency?

Reply:
It tries to get the default company currency if it is not specified in the options. This fails.
So had to call the formatter for currency while we are calling the formatters for links and dynamic links [here](https://github.com/frappe/frappe/pull/12202/files#diff-8dc4fa607cefaa232e644fad86ce669ea159a334f95aa09a0a7048537c30b3bbR239)